src/presets/netlify.ts

```diff
@@ -64,7 +64,8 @@ export const netlifyEdge = defineNitroPreset({
   },
   rollupConfig: {
     output: {
-      entryFileNames: 'server.js'
+      entryFileNames: 'server.js',
+      format: 'esm'
```

Comment:
Why is this required?

Reply:
because otherwise it does not export anything but outputs an IIFE.